test-support/build.gradle

```diff
@@ -0,0 +1,67 @@
+import org.gradle.util.GradleVersion
+
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+version = '0.18-SNAPSHOT'
```

Comment:
We don't need to version the module I'd say.

Reply:
We don't need the version if we don't publish. I'll remove it.

Reply:
Fixed and force-pushed.